corehq/apps/api/resources/v0_1.py

```diff
@@ -150,6 +150,8 @@ def obj_get_list(self, bundle, **kwargs):
         username = bundle.request.GET.get('web_username')
         if username:
             user = WebUser.get_by_username(username)
+            if not user.is_member_of(domain):
+                user = None
```

Comment:
Won't the user get blocked entirely in `RequirePermissionAuthentication(Permissions.edit_web_users)` check if they are not a member of domain?

Reply:
That permissions check applies to the person who is making the request. This check here applies to the user specified in the url with ?web_username=<username>. Previously it resulted in a response that included a confusing error message.

Reply:
Oh right, yes. Silly me.